README.md

```diff
@@ -31,3 +31,60 @@ The version number of the "bleeding edge" in `master` is always a `-SNAPSHOT` ve
+
+### Plugins for CorDapp development.
+
+- [`net.corda.plugins.cordapp`](cordapp/README.md):
+This plugin packages your CorDapp classes into a single jar file, along with
+the contants of all of the jar's dependent jars. Dependencies which are added
+to Gradle's `cordaCompile`, `cordaRuntime` and `cordapp` configurations are
+excluded from this packaging. The final jar is then signed.
+
+- [`net.corda.plugins.cordformation`](cordformation/README.rst):
+This plugin provides `Cordform` and `Dockerform` Gradle tasks for creating
+test deployments of multiple Corda nodes and their CorDapps. It also invokes
+the Network Bootstapper over the deployment for you, and provides you with a
+`runnodes` script to boot it all up afterwards.
+
+- [`net.corda.plugins.quasar-utils`](quasar-utils/README.rst):
+This plugin configures a Gradle module to use Quasar. Specifically:
+    - It allows you to specify the Maven group, version and classifier of
+the `quasar-core` artifact to use.
+    - Adds the `quasar-core` artifact, along with all of its trensitive
+dependencies, to Gradle's `cordaRuntime` configuration.
+    - Adds the `quasar-core` artifact to Gradle's `compileOnly`
+configuration without any of its transitive dependencies.
+    - Applies `quasar-core` as a Java agent to all of the module's
+`JavaExec` tasks.
+    - Applies `quasar-core` as a Java agent to all of the module's
+`Test` tasks.
+    - Provides a `quasar` Gradle extension so that you can configure
+which classes the Quasar java agent should not instrument at runtime.
+
+### Internal Corda plugins.
+These plugins are unlikely to be useful to CorDapp developers outside of R3.
+
+- [`net.corda.plugins.api-scanner`](api-scanner/README.md):
+This plugin scans the `public` and `protected` classes inside a Gradle
+module's "primary" jar artifact and writes a summary of their `public`
+and `protected` methods and fields into an output file. The "primary"
+jar is assumed by default to be the one without an `archiveClassifier`,
+although this is configurable. Its goal is to alert Corda developers to
+accidental breaks in our public ABI for those Corda modules we have
+declared to be "stable".
+
+- [`net.corda.plugins.jar-filter`](jar-filter/README.md):
+This plugin allows us to delete certain annotated classes, methods and
+fields from the compiled byte-code inside a jar file. It can also rewrite
+Kotlin classes' `@kotlin.Metadata` annotations to make them consistent 
+again with their edited byte-code. It has been successfully tested with
+Kotlin 1.2.x - 1.3.61.
+
+- [`net.corda.plugins.publish-utils`](publish-utils/README.rst): **_Here
+be Dragons!_** This plugin helps configure Gradle's `maven-publish` plugin
+to publish Corda modules to R3's Artifactory. However, its workings probably
+shouldn't be allowed and they should _most definitely_ **never** be copied.
+Unfortunately, my prior attempts at rewriting it also broke its interaction
+with JFrog's Artifactory plugin :-(.
```

Comment:
Can we reword this to be a tad more serious and dangerous sounding? Also, I really don't like emojis in docs.